front/public/locales/en/operationalStudies/project.json

```diff
@@ -0,0 +1,49 @@
+{
+  "affairCode": "Affair code",
+  "budget": "Budget",
+  "createdOn": "Created on",
+  "createStudy": "Create a study",
+  "description": "description",
+  "error": {
+    "unableToPostDocumentTitle": "Picture upload failed",
+    "unableToPostDocument": "Picture loading impossible"
+  },
+  "filterPlaceholder": "Search…",
+  "fundedBy": "Funding",
+  "geremiCode": "GEREMI code",
+  "modifiedOn": "Updated on",
```

Comment:
Might be a good time to update the key to `"updatedOn"` and `"updateProject"` ?

Reply:
yes ! done